blockly-ws-search/README.md

```diff
@@ -0,0 +1,30 @@
+[Home](../README.md)
+
+# blockly-ws-search [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+This package provides an implementation of workspace search for a Blockly workspace.
+
+## Installation
+
+```
+npm install blockly-ws-search
```

Comment:
Just a heads up I think this is going to change depending on how we decide to do naming.